db/controllers.js/controllers.js

```diff
@@ -38,6 +40,14 @@ exports.getWholeArticle = (req, res, next) => {
       .catch((err) => {
         next(err);
       });
+  } else if (typeof topics === "string") {
+    filterByTopics(topics)
```

Comment:
You shouldn't have a separate model here. There should just be one model that either gets all articles, or takes queries of `topic/sort_by/order`. I've left you some feedback on the previous sort_by/order ticket, so hopefully this makes sense in line with that!